paymentsheet/src/main/java/com/stripe/android/paymentsheet/ui/AddPaymentMethod.kt

```diff
@@ -92,7 +92,8 @@ internal fun AddPaymentMethod(
 
     Column(modifier = modifier.fillMaxWidth()) {
         CompositionLocalProvider(
-            LocalAutofillEventReporter provides sheetViewModel::reportAutofillEvent
+            LocalAutofillEventReporter provides sheetViewModel::reportAutofillEvent,
+            LocalCardNumberCompletedEventReporter provides sheetViewModel::reportCardNumberCompleted
```

Comment:
```suggestion
            LocalCardNumberCompletedEventReporter provides sheetViewModel::reportCardNumberCompleted,
```

Worth adding trailing commas to reduce future diffs.